docs/tutorials/kafka-streams-examples.rst

```diff
@@ -3,259 +3,5 @@
 Kafka Streams Examples
 ----------------------
 
-In this tutorial we will run Confluent's
-:cp-examples:`Kafka Music demo application|src/main/java/io/confluent/examples/streams/interactivequeries/kafkamusic/KafkaMusicExample.java`
-for the Kafka Streams API.
+`This content has moved. <http://docs.confluent.io/current/streams/kafka-streams-examples/docs/index.html>`_
```

Comment:
do we even want to maintain this in the `tutorials.rst` toc? I think we could remove it but this page would still be build so existing links to `/current/` would still work.